### ISSUE TITLE
Failed to login with "mxadmin" 

### ISSUE THREAD

**Issue by edwinmp** [mendix/LoginForm#16]
Failed to login because the "mxadmin" username is always converted, even when I've set the username to lower case in the Project security > Tab Administrator > Username.

```
 // local testing requires you to login with MxAdmin, which is case sensitive
if (user.toLowerCase() === "mxadmin") {
    user = "MxAdmin";
}
```
Found at:
https://github.com/mendix/LoginForm/blob/master/src/LoginForm/widget/LoginForm.js#L158

I think this bit of code should be removed, unless there is an other good reason for it.
@JelteMX Please advise.

**Comment by JelteMX** [comment]
@edwinmp Agreed, could be removed. This was done for a certain project, but is dirty

**Comment by JelteMX** [comment]
Fixed in [3.5.0](https://github.com/mendix/LoginForm/releases/tag/v3.5.0)
